### PR TITLE
IF-403: show applets tab in asset view

### PR DESCRIPTION
**Pull request by seemueller**

### CODE REVIEW

**Comment by mattmikulina** [comment]
Task linked: [IF-403 Show applets tab in Asset view](https://app.clickup.com/t/2177764/IF-403)

File: fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.html [hunk 1]
```diff
@@ -14,48 +14,59 @@
 -->
 <div class="wrapper">
   <div class="clr-row clr-justify-content-between clr-align-items-center list-header">
-    <div class="if-title">{{ rules?.length | i18nPlural: titleMapping }}</div>
-    <div *ngIf="isRouteActive('overview')">
-      <app-create-button [text]="'Create Applet'" (createItem)="createItem()"></app-create-button>
+    <div class="if-title">{{ filtedRules?.length | i18nPlural: titleMapping }}</div>
```
Comment (jsylupp) [inline fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.html:17]: typo

File: fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.scss [hunk 2]
```diff
@@ -54,3 +58,17 @@
     }
   }
 }
+
+:host ::ng-deep .p-selectbutton .p-button.p-highlight {
```
Comment (jsylupp) [inline fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.scss:62]: consider creating global style-class

File: fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.ts [hunk 3]
```diff
@@ -140,8 +152,17 @@ export class FusionAppletsOverviewComponent implements OnInit {
...
 
   private getRuleDetails() {
-    for (let i = 0; i < this.rules.length; i++) {
-      this.oispService.getRule(this.rules[i].id).subscribe(rule => this.rules[i] = rule);
+    for (let i = 0; i < this.filtedRules.length; i++) {
+      if (!this.filtedRules[i].actions) {
+        this.oispService.getRule(this.filtedRules[i].id).subscribe(rule => {
+          const ruleIndex = this.rules.findIndex(searchRule => searchRule.id === rule.id);
+          this.rules[ruleIndex] = rule;
+          const filterRule = this.filterRulesByStatus([rule])[0];
```
Comment (jsylupp) [inline fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.ts:160]: filteredRule?

File: fusionfrontend/src/app/fusion-applets/pages/fusion-applets-overview/fusion-applets-overview.component.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+export class FusionAppletsOverviewComponent implements OnInit {
+
+  constructor(
+    private activatedRoute: ActivatedRoute,
+  ) { }
+
+  ngOnInit(): void {
+  }
+
+
```
Comment (jsylupp) [inline fusionfrontend/src/app/fusion-applets/pages/fusion-applets-overview/fusion-applets-overview.component.ts:33]: remove space

**Review by jsylupp — CHANGES_REQUESTED** [review]

File: fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.scss (hunk 2 above)
Comment (seemueller) [reply fusionfrontend/src/app/fusion-applets/components/fusion-applets-list/fusion-applets-list.component.scss:62]: it is too custom

**Pull request merged**